continuous-pipe.yml.erb

```diff
@@ -220,3 +248,17 @@ tasks:
               limits:
                 cpu: 1
                 memory: 1G
+
+  migration:
```

Comment:
the discussion on using the DATABASE_ADMIN_* variables convinced me your way best to allow the migration to use them while not using them in the web service